scripts/release_check.py

```
@@ -0,0 +1,142 @@
+
+REPO = "BumpyClock/hermes"
+MODULE = "github.com/" + REPO
+VERSION = r"(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)"
+
+
+class CheckError(Exception):
+    pass
+
+
+def run(*args):
+    result = subprocess.run(args, capture_output=True, text=True, check=False)
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)